tests/basic_vm_import_test.go

```diff
@@ -14,23 +14,20 @@ import (
 	v1 "kubevirt.io/client-go/api/v1"
 )
 
-var (
-	// ID of a VM existing in oVirt described by the secret.
-	vmID = "123"
-
-	trueVar = true
-
-	diskID           = "123"
-	storageDomainsID = "123"
-
-	storageClass = "local"
-)
-
 type basicVmImportTest struct {
 	framework *fwk.Framework
 }
 
 var _ = Describe("Basic VM import ", func() {
+	var (
+		vmID = "123"
```

Comment:
What do you think about moving those constants somewhere from where we can reference them in many tests?
We could do it in different PR.

Reply:
:+1: I thought about it as well, will do it in a separate PR.